toolkits/landcover/ee_repo_sync.py

```diff
@@ -58,6 +58,12 @@ def search_replace(start_path, ext, original, replacement):
                     help='Destination path in target repo (default: root)')
 args = parser.parse_args()
 
+print('''
+    NOTE: If you receive PERMISSION_DENIED errors, be sure you have access
+    to the specified repo, and that you have logged in following instructions
+    at https://www.googlesource.com/new-password and try again.
+''')
```

Comment:
`print('foo')` is python3 style, whereas `print 'foo'` is python2.  The shebang at the top is `#!/usr/bin/env python`.  That typically resolves to python2.  I think the shebang should be changed to python3.

https://stackoverflow.com/questions/6908143/should-i-put-shebang-in-python-scripts-and-what-form-should-it-take

Reply:
SGTM. Done!